src/contour/ContourGuiApp.cpp

```diff
@@ -389,6 +389,16 @@ int ContourGuiApp::terminalGuiAction()
     // Spawn initial window.
     newWindow();
 
+    if (auto const& bell = config().profile().bell; bell == "off")
+    {
+        qmlEngine_->rootObjects().first()->findChild<QObject*>("BellAudioOutput")->setProperty("muted", true);
+    }
+    else if (bell != "default")
+    {
+        QUrl const path(bell.c_str());
+        qmlEngine_->rootObjects().first()->findChild<QObject*>("Bell")->setProperty("source", path);
```

Comment:
Can `qmlEngine_->rootObjects().first()->findChild<QObject*>("Bell")` return `nullptr`? If so, we should dereference it only if it's assured to be non-null.